spec/observables/dom/ajax-spec.ts

```diff
@@ -1127,7 +1114,13 @@ describe('ajax', () => {
 });
 
 class MockXMLHttpRequest {
-  public static readonly DONE = 4;
+  static readonly DONE = 4;
+
+  /**
+   * Set to `true` to test IE code paths.
+   */
+  static noResponseProp = false;
```

Comment:
This is that new property that is used to simulate older browsers that would lack a `response` property on their XHR.